src/Aside/MonthCalendar.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { renderWithProviders } from '../utils/testUtils';
+import MonthCalendar from './MonthCalendar';
+import { setCalendarDate } from '../store/slices/calendarDateSlice';
+
+describe('MonthCalendar Component', () => {
+  describe('when initially rendered', () => {
+    it('should display the correct month and year', () => {
+      const { store } = renderWithProviders(<MonthCalendar />);
+      const calendarDate = store.getState().calendar.calendarDate;
+      const monthYear = `${calendarDate.toLocaleString('default', { month: 'short' })} ${calendarDate.getFullYear()}`;
+      expect(screen.getByText(monthYear)).toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.getByText('2024-01-01')).toBeInTheDocument();
```

Reply:
Try to be more specific with your selectors: https://testing-library.com/docs/dom-testing-library/api-within/